assets/model_monitoring/components/src/shared_utilities/df_utils.py

```diff
@@ -34,6 +38,98 @@ def get_categorical_columns(column_dtype_map: dict) -> list:
     return categorical_columns
 
 
+def is_numerical_new(column, column_dtype_map: dict, feature_type_override_map: dict, df):
+    """Check if int column should be numerical."""
+    if feature_type_override_map.get(column, None) == "numerical":
+        return True
+    if feature_type_override_map.get(column, None) == "categorical":
+        return False
+    if column_dtype_map[column] in data_type_numerical_group:
+        return True
+    if column_dtype_map[column] in data_type_categorical_group:
+        return False
+    if column_dtype_map[column] in data_type_long_group:
+        distinct_value_ratio = get_distinct_ratio(df.select(column).rdd.flatMap(lambda x: x).collect())
```

Comment:
with override feature, do we still want to apply this logic to determine numerical/categorical column?

Reply:
I think so, because override feature is optional. If the user did not specify, it would be good to keep it

Reply:
this logic is complicated and yet not guarantee it is what user want, IMO, if no feature override, maybe just simply determine the numerical/categorical by dtype. You can discuss with Yunjie and Feng to see if we want to keep it.

Reply:
but for the integer type, we cannot know if it is numerical or categorical. I think that is the common design for determining the integer type?

Reply:
Checked with Yunjie. Keep the current implementation